components/dialogs/AuthDialog.vue

```diff
@@ -90,7 +109,16 @@
           key="portal"
           class="auth-dialog__tab auth-dialog__tab--index"
         >
-          <signin-portal @submit="signInWithPlatform" />
+          <Transition
+            name="fade"
+            mode="out-in"
+          >
+            <signin-portal
+              :key="isSignIn.toString()"
```

Comment:
So the key is `"true" / "false"`?

Reply:
Yes, since string seems more intuitive as key